tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -507,35 +533,46 @@ protected void CopyToFast(ref MemoryPoolIterator output)
                         }}
                     }}
 
+                    if({header.TestNotTempBit()})
```

Comment:
Do we know this is actually faster than doing the `tempBits &= ~{1L << header.Index}L;` before the if condition and then comparing `tempBits` to zero. If so, why are you doing it the old way just for the Content-Length header?